samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -223,23 +223,10 @@ static void update_shadow(void)
 		.fota = &fota_info,
 		.ui = &ui_info
 	};
-#if defined(CONFIG_NRF_MODEM)
-	struct nrf_cloud_modem_info modem_info = {
-		.device = NRF_CLOUD_INFO_SET,
-		.network = NRF_CLOUD_INFO_SET,
-		.sim = IS_ENABLED(CONFIG_MODEM_INFO_ADD_SIM) ? NRF_CLOUD_INFO_SET : 0,
-		.mpi = NULL,
-		/* Include the application version */
-		.application_version = CONFIG_APP_VERSION
-	};
-	struct nrf_cloud_modem_info *mdm = &modem_info;
-#else
-	struct nrf_cloud_modem_info *mdm = NULL;
-#endif
 	struct nrf_cloud_device_status device_status = {
-		.modem = mdm,
+		.modem = NULL,
```

Comment:
Right, it's only for clarity -- I understand that if you don't specify a given struct element in the initializer, it will be set to 0.  Purists will argue that NULL might be something other than 0 on some architectures (though that seems unusual to me).

This section is a bit confusing though.  Here, the modem info is not updated in the shadow, which makes one wonder why not.  It turns out the automatic shadow updates done in `nrf_cloud_state_encode()` will have already done so.

But for CoAP, the equivalent is done in `nrf_cloud_coap_connect()`.

Reply:
```suggestion
/* Modem info is sent automatically since CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS is enabled, so it can be skipped here. */
		.modem = NULL,
```

perhaps such a comment would lessen the confusion?
also i think georges created a ticket to rename "modem info" and potentially re-organize this status struct?  
that should make things more clear.

Reply:
Yes, I was going to do so, but you beat me to it.